### DISCUSSION TITLE
Device stop working. Help please.

### DISCUSSION THREAD

**Discussion by bananajoe86** [arendst/Tasmota#14502] [Q&A] [answered]
### PROBLEM DESCRIPTION
I used a lot of Tasmota-Devices, but one makes me trouble. Tasmota stop working after a time (between one or two days). The Wemos D1 Mini isn´t working local (physical botton) or can be accessed by interface. I can solve the problem if I poweroff the device several times. 6x Powerless does not reset the controller, but I can get it running again with the configuration. A Software-Restart is everytime possible (if the controller is online)

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): Wemos D1 mini
- [x] Tasmota binary firmware version number used: 10.1.0
  - [ ] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [x] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:
18:06:43.954 MQT: stat/Wemos-Dunstabzug/RESULT = {"NAME":"Generic","GPIO":[1,1,1,1,1,1,1,1,1,1,1,1,1,1],"FLAG":0,"BASE":18}
18:06:44.169 MQT: stat/Wemos-Dunstabzug/RESULT = {"Module":{"0":"Generic"}}
18:06:44.425 MQT: stat/Wemos-Dunstabzug/RESULT = {"GPIO0":{"227":"Relay4"},"GPIO1":{"224":"Relay1"},"GPIO2":{"162":"Switch3"},"GPIO3":{"161":"Switch2"},"GPIO4":{"257":"Relay_i2"},"GPIO5":{"160":"Switch1"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO12":{"640":"I2C SDA"},"GPIO13":{"608":"I2C SCL"},"GPIO14":{"1184":"DHT11"},"GPIO15":{"226":"Relay3"},"GPIO16":{"0":"None"},"GPIO17":{"0":"None"}}

```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [x] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:
  18:07:55.267 CMD: status 0 
18:07:55.276 MQT: stat/Wemos-Dunstabzug/STATUS = {"Status":{"Module":0,"DeviceName":"Wemos-Dunstabzug","FriendlyName":["Tasmota","","",""],"Topic":"Wemos-Dunstabzug","ButtonTopic":"0","Power":8,"PowerOnState":3,"LedState":0,"LedMask":"FFFF","SaveData":0,"SaveState":1,"SwitchTopic":"0","SwitchMode":[4,4,1,0,0,1,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0}}
18:07:55.286 MQT: stat/Wemos-Dunstabzug/STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"https://github.com/arendst/Tasmota/releases/download/v9.5.0/tasmota-sensors.bin.gz","RestartReason":"Software/System restart","Uptime":"0T00:17:17","StartupUTC":"2022-01-18T16:50:38","Sleep":50,"CfgHolder":4617,"BootCount":1379,"BCResetTime":"2021-08-06T13:55:27","SaveCount":1423,"SaveAddress":"F5000"}}
18:07:55.294 MQT: stat/Wemos-Dunstabzug/STATUS2 = {"StatusFWR":{"Version":"10.1.0(sensors)","BuildDateTime":"2021-12-08T14:47:41","Boot":31,"Core":"2_7_4_9","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8266EX","CR":"403/699"}}
18:07:55.302 MQT: stat/Wemos-Dunstabzug/STATUS3 = {"StatusLOG":{"SerialLog":2,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["FRITZ!7390",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["00008009","2805C80001000600003C5A0A000000000000","000000C0","00006000","00004000"]}}
18:07:55.321 MQT: stat/Wemos-Dunstabzug/STATUS4 = {"StatusMEM":{"ProgramSize":668,"Free":332,"Heap":19,"ProgramFlashSize":1024,"FlashSize":4096,"FlashChipId":"16400B","FlashFrequency":40,"FlashMode":3,"Features":["00000809","8FDA8787","0415A005","B7F7BFCF","05DA9BC4","64367CC7","00084052","20000000","00000020"],"Drivers":"1,2,3,4,5,6,7,8,9,10,12,14,16,17,20,21,24,29,34","Sensors":"1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,17,18,19,20,21,22,26,28,31,34,37,39,40,42,43,45,51,52,55,56,58,59,62,64,66,67,74"}}
18:07:55.331 MQT: stat/Wemos-Dunstabzug/STATUS5 = {"StatusNET":{"Hostname":"Wemos-Dunstabzug","IPAddress":"192.168.178.162","Gateway":"192.168.178.1","Subnetmask":"255.255.255.0","DNSServer1":"192.168.178.102","DNSServer2":"0.0.0.0","Mac":"24:A1:60:2F:C3:21","Webserver":2,"HTTP_API":1,"WifiConfig":4,"WifiPower":20.0}}
18:07:55.342 MQT: stat/Wemos-Dunstabzug/STATUS6 = {"StatusMQT":{"MqttHost":"192.168.178.103","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_2FC321","MqttUser":"DVES_USER","MqttCount":1,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30,"SOCKET_TIMEOUT":4}}
18:07:55.353 MQT: stat/Wemos-Dunstabzug/STATUS7 = {"StatusTIM":{"UTC":"2022-01-18T17:07:55","Local":"2022-01-18T18:07:55","StartDST":"2022-03-27T02:00:00","EndDST":"2022-10-30T03:00:00","Timezone":"+01:00","Sunrise":"08:36","Sunset":"17:24"}}
18:07:55.367 MQT: stat/Wemos-Dunstabzug/STATUS10 = {"StatusSNS":{"Time":"2022-01-18T18:07:55","Switch1":"OFF","Switch2":"OFF","Switch3":"OFF","DHT11":{"Temperature":25.0,"Humidity":41.0,"DewPoint":10.8},"TempUnit":"C"}}
18:07:55.381 MQT: stat/Wemos-Dunstabzug/STATUS11 = {"StatusSTS":{"Time":"2022-01-18T18:07:55","Uptime":"0T00:17:17","UptimeSec":1037,"Heap":20,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"POWER1":"OFF","POWER2":"OFF","POWER3":"OFF","POWER4":"ON","Wifi":{"AP":1,"SSId":"FRITZ!7390","BSSId":"74:42:7F:06:A6:5F","Channel":6,"Mode":"11n","RSSI":68,"Signal":-66,"LinkCount":1,"Downtime":"0T00:00:07"}}}


```
- [x] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:
18:08:45.556 CMD: weblog 4
18:08:45.563 MQT: stat/Wemos-Dunstabzug/RESULT = {"WebLog":4}
18:08:52.687 WIF: Checking connection...
18:09:12.673 WIF: Checking connection...
18:09:32.699 WIF: Checking connection...
18:09:40.646 WIF: Sending Gratuitous ARP
18:09:52.715 WIF: Checking connection...
18:10:07.681 DHT: Timeout waiting for start signal low pulse
18:10:12.759 WIF: Checking connection...
18:10:15.664 DHT: Timeout waiting for start signal low pulse
18:10:23.676 DHT: Timeout waiting for start signal low pulse
18:10:31.677 DHT: Timeout waiting for start signal low pulse
18:10:32.710 WIF: Checking connection...
18:10:39.656 WIF: Sending Gratuitous ARP
18:10:39.679 DHT: Timeout waiting for start signal low pulse
18:10:47.670 DHT: Timeout waiting for start signal low pulse
18:10:52.713 WIF: Checking connection...

```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
Configuration
![image](https://user-images.githubusercontent.com/52297141/149981684-2dab4b0c-bb8e-4c93-968a-f4005f3eeefc.png)

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by Jason2866** [comment]
Without a good crystal ball (we dont have found a source for), it is hard to find the reason. You did not mentioned details nor the issue template is filled.

**Comment by sfromis** [comment]
With 3 relays, you have to make sure that your power source is up to the task, and that you have done appropriate wiring for relays, which can draw quite a bit of power (in the context of ESP8266).

**Comment by bananajoe86** [comment]
I'm trying to provide as much information as possible to help narrow down the problem. if something is missing please let me know.
The power source should be okay (3A). As long as the device does not go into blackout, it works perfectly with all relays and sensors. For the momenten I haven´t andy idea, why it goes offline regularly and not online again with only one power off.

**Comment by Jason2866** [comment, marked as answer]
Please provide the wiring / circuit diagram. How are the relais connected and powered?
